sound/soc/sof/sof-priv.h

```diff
@@ -105,6 +105,11 @@ enum sof_debugfs_access_type {
 	SOF_DEBUGFS_ACCESS_D0_ONLY,
 };
 
+struct sof_compr_stream {
+	uint32_t copied_total;
```

Comment:
if this is internal to kernel, the type should be u32. 

Actually it should be u64, so that we are ready for a future change where the compressed timestamping works even with long tracks, it's broken currently with a limitation to u32.

Reply:
@plbossart all right, i used u32 mimicking the compressed timestamp :). will fix.